tests/data_files/test_file

```diff
@@ -0,0 +1 @@
+signature test
```

Comment:
Please give this file and the associated files a less generic file name. Why is all this data in separate files anyway? Why isn't it directly in `test_suites_rsa.data`? Where is this file used anyway?

Reply:
@gilles-peskine-arm My intention is to include documentation for each new piece of test data on how it was generated. I first generated the files `test_file-*` as documented in `test_file-Readme.txt` and then copied their contents to `test_suite_rsa.data`, as also mentioned in the Readme. It would perhaps be more appropriate to make that Readme an extended comment in the test suite data file, but such are currently not allowed, unfortunately.

Reply:
@gilles-peskine-arm What's your suggestion on that? I don't like just throwing the files and documentation away, because then we're again in the situation where we have some chunk of test data nobody knows where it's coming from and what it's supposed to trigger. I agree, though, that the current solution is not optimal either.

Would you accept keeping the files if they are given a more descriptive name?

Reply:
@hanno-arm  I'm not too fond of duplicating the data. I think the easiest solution is to allow comments in test data files. PR #1116

Reply:
@gilles-peskine-arm Thanks alot, I've removed the test files and the readme and instead added comments to the RSA test suite.